### DISCUSSION TITLE
Track waypoint proximity alerts - still available?

### DISCUSSION THREAD

**Discussion by barteis** [osmandapp/OsmAnd#11471] [Q&A]
In an earlier version of OsmAnd, I was able to get "proximity alerts" when approaching track waypoints.  (See attached)  I'm unable to do that with v. 3.9.  Is the feature gone?  Am I missing a setting? The feature does seem to work using Favorites.  Thanks.  

![track waypont alert](https://user-images.githubusercontent.com/36173950/115276194-97dad780-a0f7-11eb-91b2-d23a9181dbb3.png)

**Comment by vshcherb** [comment]
Should be still working recently was tested with voice alerts, the issue was missing in 3.8 and got back in 3.9

**Reply by barteis** [reply]
Using version 3.9.10, I re-tested navigating with two GPX routes, both created with Plan a route.  For each GPX, I added both track waypoints and some nearby Favorites.  When I got near a favorite, it was announced and I saw an alert at the top of the screen.  When I got near a track waypoint, I got neither--no announcement, no screen alert.
